pkg/controller/operator.go

```diff
@@ -120,14 +119,11 @@ func (s *Operator) Run(ctx context.Context, controller *controllercmd.Controller
 	rec := recorder.New(recdriver, s.Interval, anonymizer)
 	go rec.PeriodicallyPrune(ctx, statusReporter)
 
-	// the gatherers periodically check the state of the cluster and report any
-	// config to the recorder
-	clusterConfigGatherer := clusterconfig.New(
+	// the gatherers are periodically called to collect the data from the cluster and write the results to the recorder
+	importantGatherers, optionalGatherers := gather.CreateAllGatherers(
```

Comment:
I think the naming `importantGatherers` and `optionalGatherers` is not that clear and can be a litle bit misleading. I would call them for example `clusterConfigGatherer` and `workloadGatherer`. WDYT?

Reply:
Yeah, naming could be better, but the idea is that we can add more than 2 gatherers later.

Reply:
changed it